specification/syntex/resource-manager/Microsoft.Syntex/preview/2022-09-15-preview/syntex.json

```diff
@@ -0,0 +1,493 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-09-15-preview",
+    "title": "Syntex",
+    "description": "Syntex provider"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Syntex/documentProcessors": {
+      "get": {
+        "description": "Returns document processors in the subscription.",
+        "operationId": "DocumentProcessor_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Document processor details.",
+            "schema": {
+              "$ref": "#/definitions/DocumentProcessorList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "DocumentProcessor_List": {
+            "$ref": "./examples/DocumentProcessor_List.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Syntex/documentProcessors": {
+      "get": {
+        "description": "Returns document processors in a resource group.",
+        "operationId": "DocumentProcessor_ListByResourceGroup",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
```

Comment:
I may not have been clear in my earlier comemnt but if possible I'd recommend just referencing the latest parameter defnitions from common-types instead i.e.

"$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"

"$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"

"$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"